packages/plugins/rudder_plugin_ios/ios/Classes/RudderSdkFlutterPlugin.m

```diff
@@ -1,12 +1,14 @@
 #import "RudderSdkFlutterPlugin.h"
 #import <Rudder/Rudder.h>
 #import "RSMessageType.h"
+#import "RSLogger.h"
 
 static NSNotification* _notification;
 
 @implementation RudderSdkFlutterPlugin
 
 NSMutableArray* integrationList;
+BOOL _isCleanUp = NO;
```

Comment:
I think we should call it `isRegistrarDetached`
Can you move these two variables to `.h` file?
@interface RudderSdkFlutterPlugin : NSObject <FlutterPlugin> {
    NSMutableArray* integrationList;
    BOOL isRegistrarDetached;
}

Reply:
Sure will do it